tests/test_Matcher.py

```diff
@@ -0,0 +1,80 @@
+from os import remove
+from prioritizator import Matcher
+from prioritizator import Task
+from shutil import copyfile
+
+WHITE_WINS = 1
+DRAW = 0.5
+WHITE_LOSES = 0
+
+
+def test_constructor_0_arguments():
+    matcher = Matcher()
+    obtained_data_path = matcher.data_path
+    expected_data_path = None
+    assert obtained_data_path == expected_data_path
+
+
+def test_setting_data_path():
+    data_path = "tests/test_data/test_load_task_list.csv"
+    matcher = Matcher()
+    matcher.data_path = data_path
+    obtained_data_path = matcher.data_path
+    expected_data_path = data_path
+    assert obtained_data_path == expected_data_path
+
+
+def test_constructor_1_argument():
+    data_path = "tests/test_data/test_load_task_list.csv"
+    matcher = Matcher(data_path)
+    obtained_data_path = matcher.data_path
+    expected_data_path = data_path
+    assert obtained_data_path == expected_data_path
+
+
+def test_match_existing_tasks():
+    data_path = "tests/test_data/test_save_task_list.csv"
+    copyfile("tests/test_data/test_load_task_list.csv", data_path)
+    white_id = 1
+    white_task = Task(id=white_id).load_from_csv(data_path)
+    obtained_white_rating_before = white_task.rating
+    expected_white_rating_before = 1000
+    assert expected_white_rating_before == obtained_white_rating_before
```

Comment:
Estas líneas se repiten con las 62-67, por favor refactorízalas en una nueva función. Tal vez podría quedar algo así:
```suggestion
def assert_white_rating_before(white_id, expected_white_rating_before):
    white_task = Task(id=white_id).load_from_csv(data_path)
    obtained_white_rating_before = white_task.rating
    expected_white_rating_before = 1000
    assert expected_white_rating_before == obtained_white_rating_before
```

Reply:
Me imagino algo asi:
```python
def assert_white_rating_before(white_id, expected_white_rating_before):
    white_task = Task(id=white_id).load_from_csv(data_path)
    obtained_white_rating_before = white_task.rating
    assert expected_white_rating_before == obtained_white_rating_before
```
Pero hay un pequeño inconveniente, y este es que dentro de la misma prueba quedaria un assert que al final requeriria un valor que se obtendria dentro de esta nueva funcion:
Linea 55
`    assert obtained_white_rating_after < obtained_white_rating_before
`
Lo que me hace pensar, podemos regresar este valor al contexto original de la prueba para que continue con su flujo?
__NOTA__:Esto mismo ocurre en dos pruebas distintas pero similares
- test_match_existing_tasks()
- test_match_new_tasks()

definitivamente debemos seguir rebotando ideas, aqui hay oportunidad.